src/scss/_menulog.scss

```diff
@@ -11,39 +11,60 @@
 // =================================
 
 @mixin applyScheme-menulog {
-    $green                        : #028a00 !global;
-    $green--dark                  : #0f550c !global;
-    $green--darkest               : #003c00 !global;
-    $green--light                 : #f2fae2 !global;
-    $green--offWhite              : #dfffdf !global;
-
-    $red--light                   : #fff1df !global;
 
     // Brand specific colours
-    // Not accessible in all circumstances but available if needed in certain designs
-    $brand--green                 : #26bf22 !global;
+    // NOT AA accessible but available for elements and backgrounds that don't need to be accessible
+    $brand--orange                : #ff8000 !global;
```

Comment:
Is the plan to eventually move these colours into the main SCSS file if the colour changes are rolling out to JE as well?

Reply:
Yep – that's why I've made this a beta, as the next few updates to this package will all be breaking changes 😆 

When we've updated the default theme, I would think there will just be one theme – should be able to get rid of `ml` hopefully